bors.toml

```
@@ -1,5 +1,6 @@
 status = [
-  "ci/hydra:Cardano:cardano-prelude:required",
+  "ci",
+  "ci (x86_64-linux)"
```

```suggestion
  "ci"
```

With the latest tullia version `ci` can only pass if `ci (x86_64-linux)` and all other available platforms also pass so just `ci` is sufficient if you want to require all platforms to pass. If you only want to require linux, list only `ci (x86_64-linux)` without `ci`.